schema/2.0.0/corpora_schema.md

```diff
@@ -0,0 +1,262 @@
+
+* **Organisms**. Data MUST be either Human or Mouse data. No other organisms are accepted by the cellxgene Data Portal. 
+
+* **Name Collisions in Metadata**. If a metadata key exists in the original dataset with the exact name required by the schema but with values that cannot be accurately mapped to the schema, `"key"` MUST be renamed to `"key_original"`. For example, if there is a `"cell_type"` in the original dataset with values that cannot be mapped, then it MUST be renamed to `"cell_type_original"`. 
+
+* **Redundant Metadata**. It is STRONGLY RECOMMENDED to avoid multiple metadata fields containing identical or similar information. 
+
+*   **No PII**. Curators agree to this requirement as part of the data submission policy.
+    However, it is not strictly enforced in our validation tooling because it is difficult for software to predict what is and is not PII.
+    It is up to the submitter to ensure that no metadata can be personally identifiable: no names, dates of birth, specific locations, etc.
+    See this [list](https://docs.google.com/document/d/1sboOmbafvMh3VYjK1-3MAUt0I13UUJfkQseq8ANLPl8/edit) for guidance.
+
+#### *Note on types*
+The types below are python3 types. Note that a python3 `str` is a sequence of Unicode code points, which is stored null-terminated and UTF-8-encoded by anndata.
+
+## `X` (Matrix Layers)
+
+cellxgene does not impose any additional constraints on the `X` data matrix.
+
+In any layer, if a matrix has 50% or more values that are zeros, it is STRONGLY RECOMMENDED that the matrix be encoded as a [`scipy.sparse.csr_matrix`](https://docs.scipy.org/doc/scipy/reference/generated/scipy.sparse.csr_matrix.html).
+
+cellxgene's matrix layer requirements are tailored to optimize data reuse. Because each assay has different characteristics, the requirements differ by assay type. In general,
+cellxgene requires submission of "raw" data suitable for computational reuse when a standard raw matrix format exists for an assay and strongly recommends that a "final" matrix suitable for
+visualization in the explorer be included. So that cellxgene's data can be provided in download formats suitable for both R and Python, the schema imposes the following requirements:
+
+*   All matrix layers MUST have the same shape, and have the same cell labels and gene labels.
+*   Because it is impractical to retain all barcodes in raw and final matrices, any cell filtering MUST be applied to both.
+    By contrast, those wishing to reuse datasets require access to raw gene expression values, so genes MUST NOT be filtered from either dataset.
+    Summarizing, any cell barcodes that are removed from the data MUST be filtered from both raw and final matrices and genes MUST NOT be filtered from the raw matrix.
```

Comment:
>genes MUST NOT be filtered from the raw matrix.

How will this be validated?

Reply:
@ambrosejcarr - is this a STRONGLY RECOMMENDED rather than a MUST?

Reply:
> @ambrosejcarr - is this a STRONGLY RECOMMENDED rather than a MUST?

Yes. We could design some heuristics to catch cases where this is obviously broken but they would be assay-specific and probably more trouble than they're worth.

Reply:
Addressed in https://github.com/chanzuckerberg/single-cell-curation/pull/67/commits/3de2a989d506c00a6b8374f8a92b6c5308dbc50e